internal/connectors/db/process_result_set.go

```diff
@@ -50,7 +88,7 @@ func ReadOperationFromResultSet(res result.Result) (types.Operation, error) {
 	}
 	if operationType == types.OperationTypeTB {
 		if backupId == nil || database == nil || ydbOperationId == nil {
-			return nil, fmt.Errorf("failed to read required fields of operation %s", operationId.String())
+			return nil, errors.New(fmt.Sprintf("failed to read required fields of operation %s", operationId.String()))
```

Comment:
What's the point of using `errors.New(fmt.Sprintf())` instead of `fmt.Errorf()` ?